frame/support/src/traits/validation.rs

```diff
@@ -244,3 +244,17 @@ pub trait ValidatorRegistration<ValidatorId> {
 	/// module
 	fn is_registered(id: &ValidatorId) -> bool;
 }
+
+/// Trait used to check whether a given validator is currently disabled and should not be
+/// participating in consensus (e.g. because they equivocated).
+/// The [Session module](../../pallet_session/index.html) is an implementor.
```

Comment:
```suggestion
```

This would be a dead link on doc.rs for example. And if you write who is the implementor, you not really need this trait :P